specification/2.0/Specification.adoc

```diff
@@ -1033,34 +1033,15 @@ Only the following three accessor configurations require padding.
 
 *MAT2, 1-byte components*
 
-|===
-|00|01|02|03|04|05|06|07
-|m00|m10|&#8212;|&#8212;|m01|m11|&#8212;|&#8212;
-|===
+image:figures/padding-mat2-1byte.png[pdfwidth=4in,align=left]
```

Comment:
As the first 2 images are half the width of the last, I would recommend `pdfwidth=2in` for the smaller images.

Reply:
I had searched for that `pdfwidth`, and seen that _all_ images used the same value ... but will update this (together with the attempt to directly use SVG).

Reply:
Yeah, I made that comment after seeing the result in the PDF.

In case you don't know this already, since these changes are happening in your own javagl fork, the GitHub actions are running there:

https://github.com/javagl/glTF/actions

Click on the topmost build, and find "spec-outputs" at the bottom of the summary page for the CI.  It will be a ZIP file with the built version of the PDF and HTML versions.